source/gra3-uni4/gra3-uni4-secD-lec20-act1.ptx

```diff
@@ -0,0 +1,124 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<activity xmlns:xi="http://www.w3.org/2001/XInclude"> 
+
+<title>Formas de dividir</title>
+
+<statement>
+  <ol>
+    <li><p>Lin, Priya y Tyler encontraron el valor de <m>78 \div 3</m>. Este es su trabajo. Dale sentido al trabajo de cada estudiante.</p>
+    
+    <p>Lin</p>
+    
+    <p><image source="svg-source/tikz-file-149344">
+      <shortdescription>Bloques en base 10. 7 decenas y 8 unidades. Se dividen en 3 grupos iguales </shortdescription>
+    </image></p>
+    <sidebyside>
+      <stack>
+        <p>Priya</p>
+        <p>
+          <line><m>3\times 10 = 30</m></line>
+          <line><m>3\times 10 = 30</m></line>
+          <line><m>3\times \phantom{0}6 = 18</m></line>
+          <line><m>\overline {3 \times 26 =78}</m></line>
+        </p>
```

Comment:
Creo que esto se debe hacer con el equivalente a un eqnarray, para que los `=` queden verdaderamente alineados. 
En el output HTML se ve perfecto porque todo alinea perfecto en este caso, pero mejor hacerlo con la sintaxis correcta para esto.

¿Quiere intentar? (puede que no funcione!) Ver: [Subsection 4.9.8: Alignment in Display Mathematics](https://pretextbook.org/doc/guide/html/topic-mathematics.html#topic-mathematics-11) 

Se vería así 
```xml
<md>
    <mrow><m>3\times 10 \amp = 30</m></mrow>
    <mrow><m>3\times 10 \amp = 30</m></mrow>
    <mrow><m>3\times \phantom{0}6 \amp = 18</m></mrow>
    <mrow><m>\overline {3 \times 26 \amp =78}</m></mrow>
</md>
```
(pero puede que esto no funcione por la línea del \overline!)

Reply:
Ah ok. El estilo de <line> lo copié de una lección previa que le vi (https://im.kendallhunt.com/k5_es/teachers/grade-3/unit-4/lesson-10/lesson.html), pero en esa no son ecuaciones, entonces creo que ok como lo tiene, vd?

Reply:
Ya intenté, pero no parece funcionar. Queda todo en blanco. Pero podemos seguir intentando...

Reply:
Tiene razón @aguilulfo , en algún punto hice eso con los <line>. Los <md> centran el contenido (igual que un `align` en LaTeX, entonces de pronto dejarlo como está.  

Entonces dejemos así, pero por si sirve a futuro, este es un código que sí funciona: (pero no me convence porque queda centrado en cada columna)
```xml
<sidebysidegroup>
      <sidebyside>
        <p>Priya</p>
        <p>Tyler</p>
      </sidebyside>
      <sidebyside>
        <p><md>
          <mrow>3\times 10 \amp = 30</mrow>
          <mrow>3\times 10 \amp = 30</mrow>
          <mrow>3\times \phantom{0}6 \amp = 18</mrow>
          <mrow>\hline 3 \times 26 \amp =78</mrow>
        </md></p>
        <p><md>
          <mrow>3\times 20 \amp = 60</mrow>
          <mrow>3\times \phantom{0}6 \amp = 18</mrow>
          <mrow></mrow>
          <mrow>20 + 6 \amp = 26</mrow>
        </md></p>
      </sidebyside>
    </sidebysidegroup>
```